subscribed_topics:
  input_topic: /kinect/point_cloud
  #input_topic: /camera/depth_registered/points
  unlock_topic: unlock_rgb_depth_synchronizer

published_topics:
  point_cloud_internal_topic: synchronized/depth/points
  depth_image_topic: synchronized/depth/image
  rgb_image_topic: synchronized/rgb/image
